### DISCUSSION TITLE
Turkish Translations

### DISCUSSION THREAD

**Discussion by salihdev0** [vuejs-translations/guidelines#30] [Ideas]
Hello, I've been writing Vue for about 1 year (this time may seem short, but if you see it as a 20-year-old teenager with 2 years of experience working 16+ hours a day, it's really long.). I am happy to write Vue. In Turkey, Vue is relatively less used than its closest competitor, React. When viewed, Vue does not have a Turkish document yet, but React has a Turkish document. Why shouldn't Vue have a Turkish document? I live in Turkey and I really use Vue in advanced projects. If you give the necessary authorization and time, I can write the Turkish document of Vue. At the same time, if the developers who support this article give their support, we can complete it in a shorter time!

Turkey is a country with a population of 82 million. Although there are fewer people compared to countries such as China, America and India, the percentage of software developers is high in the population. Turkey and I deserve a chance. Say what?

**Comment by mcelikx** [comment]
Hi, i can also work for translations. I am working as a frontend developer and a company owner in Turkey. I see that there are not enough Vue developers in Turkey. I think English Documentation of Vue is better than React Documentation. But lack of Turkish version for Vue make other technologies more popular. 

Turkish developers have a very good reputation around the world. However, almost all of them started to work with React at first due to the existence of Turkish documentation and they continue in this way.

I am ready to prepare Turkish documentation for Developers in Turkey. So we can give them a change to start their working life with Vue. I think a great community support will come after us.

**Comment by necatikcl** [comment]
As a student of English philology, I would be glad to contribute too if there is a chance.

**Comment by eyupucmaz** [comment]
Hi! I would love to help this community. I'am a frontend developer. Also using Vue in my professional works. When i started learning Vue after React, I expect Vue has a Turkish translations documents. If there was it would helped me learn faster. Many new developer who want to learn a framework will look for Turkish Doc. Hope there will be many developer who like to help to translate.

**Comment by anilsakadev** [comment]
Hi, I 'm a backend developer. I would also like to help with the Turkish documentation of Vue JS

**Comment by salihdev0** [comment]
For something to happen, it takes the first step, and that's what I did. Looking forward to your PRs to contribute!

📍[Repo](https://github.com/ssibrahimbas/vue-docs-tr)

**Reply by halillusion** [reply]
Repo is unavailable.

**Comment by fulutas** [comment]
Hello, would love to help this community.  I would also like to help with the Turkish documentation of Vue JS

**Comment by devisasari** [comment]
Hi! I'm Front-End Developer. I'd love to help this community and I'd like to contribute to the Vue JS Turkish documentation.

**Comment by FerhatACAR** [comment]
Hello, I have been a full-stack developer for 4 years. I've been considering contributing to translation work. If this process continues, I would like to join you guys.

**Comment by bayrameker** [comment]
Dear Vue.js community,

I hope this text finds you well. My name is Bayram, and I am writing to express my interest in volunteering for the translation of Vue.js documentation.

I have over 4 years of experience in software development, with a strong focus on web development. During this time, I have worked extensively with Vue.js and have gained a deep understanding of its concepts and best practices. In addition, I regularly contribute articles on Vue.js to Medium, where I share my knowledge and insights with the developer community.

I am fluent in both English and Turkish, and I have a proven track record of delivering high-quality translations. I am confident in my ability to accurately convey technical concepts while maintaining the original meaning and style of the documentation.

I would be thrilled to contribute to the Vue.js community by translating the documentation into Turkish. I believe that making the documentation more accessible to a wider audience will greatly benefit developers in Turkish-speaking countries and contribute to the growth and adoption of Vue.js.

Please let me know if there are any specific guidelines or requirements for the translation process. I am open to collaborating with your team and following any established workflows or using translation tools if necessary.

Thank you for considering my application. I look forward to the opportunity to contribute to the Vue.js project and help developers in Turkish better understand and utilize this powerful framework.

I have attached my resume and relevant writing samples for your reference. Please feel free to contact me if you have any further questions or require additional information.

Thank you for your time and consideration.

Sincerely,
Bayram Eker

https://medium.com/@bayramblog
https://www.linkedin.com/in/bayrameker/

**Comment by cbugk** [comment]
New dictionary dropped-in: [100. Yıl Bilişim Terimleri Sözlüğü](https://bilisimde.ozenliturkce.org.tr/docs/sozluk_S10.pdf).

https://github.com/cbugk/vuejs-docs-tr
I will commit (pun intended) to checking any merge request's conformity to the dictionary. Let's get some puritanical action going, stick to the canon.

Rule set:
* Do not send your CV or code, just translations.
* If you are a Grammar-Nazi you are in, no `chicken-translate`.
* If you have a Github account you are in. High schoolers are welcome.
* Mention @cbugk in this thread and I shall answer when I see it.

I will try to be indifferent to people's choice of using either one of Turkish or English to communicate. Sorry can't speak other languages, stick to these two please.

**Reply by exrem** [reply]
hai~

i'll try to help you to build one, sending some pull reqs in my leisure. thank you so much for engaging in! ^^

**Reply by cbugk** [reply]
@exrem I added you as a collaborator. I will not be translating for a while, trying to wrap my head around Go+HTMX currently. Take your time with translations, but as soon as they drop I will make time for review.

Shout out to others! This is my first time organizing a repository, so any help would be much appreciated. Also if @9ssi7 's repo can somehow get revived, I am okay with building on top of it. Just give me a heads up in here either way.

**Reply by exrem** [reply]
thank you so much @cbugk! :3

**Comment by salihdev0** [comment]
Hello everyone again, it's been a long time since I started this flow and the repo I mentioned in the first place has been closed. We can continue [from here](https://github.com/orgs/vuejs-translations/discussions/30#discussioncomment-7996895). Also, if there is anything I can do, I would be honored to do it.

**Comment by salihdev0** [comment]
People who want to do something can select an issue from [this section](https://github.com/cbugk/vuejs-docs-tr/issues) and start working after forking the project. Don't forget to comment on the issues you started! @cbugk will assist in the management of the project.

https://github.com/cbugk/vuejs-docs-tr/issues

**Comment by Kripteks** [comment]
Hello, I follow the issue closely.

**Reply by cbugk** [reply]
@Kripteks, I have decided not to perform translations myself as it was way more time consuming than I anticipated. But I still am willing to proof-read anyone's contributions. I would be okay with following another account's repository if someone else is willing to keep in sync with the official docs. If you have any translations please hit me up.

**Comment by AloisSeckar** [comment]
New effort to create Turkish translation was recently started - check https://github.com/vuejs/docs/discussions/3385
